config/plugins/vite.md

```diff
@@ -68,19 +68,19 @@ module.exports = {
             "build": [
               {
                 "entry": "src/main.js",
-                "config": "vite.main.config.mjs",
+                "config": "vite.main.config.mjs"
               },
               {
                 "entry": "src/preload.js",
-                "config": "vite.preload.config.mjs",
+                "config": "vite.preload.config.mjs"
               },
```

Comment:
good catch on the JSON errors, should the comma on this line be removed as well? I'm getting an error on local with this `Trailing commajson(519)`

Reply:
will be even better! done